src/policy/fees.h

```diff
@@ -199,25 +193,23 @@ static const double SUFFICIENT_FEETXS = 1;
 /** Require only an avg of 1 tx every 5 blocks in the combined pri bucket (way less pri txs) */
 static const double SUFFICIENT_PRITXS = .2;
 
-// Minimum and Maximum values for tracking fees and priorities
+// Minimum and Maximum values for tracking fees
 static const double MIN_FEERATE = 10;
 static const double MAX_FEERATE = 1e7;
 static const double INF_FEERATE = MAX_MONEY;
-static const double MIN_PRIORITY = 10;
-static const double MAX_PRIORITY = 1e16;
-static const double INF_PRIORITY = 1e9 * MAX_MONEY;
 
-// We have to lump transactions into buckets based on fee or priority, but we want to be able
-// to give accurate estimates over a large range of potential fees and priorities
+// We have to lump transactions into buckets based on fee, but we want to be able
+// to give accurate estimates over a large range of potential fees
 // Therefore it makes sense to exponentially space the buckets
 /** Spacing of FeeRate buckets */
 static const double FEE_SPACING = 1.1;
 
-/** Spacing of Priority buckets */
-static const double PRI_SPACING = 2;
+// TODO: determine which version to remove priority
+// Client version at which priority is removed
+static const int REMOVE_PRIORITY_VERSION = 0;
```

Comment:
The next release will be 3.1.0, so set this to `3.1.0-beta1`, which is `3010000`